pkg/koordlet/util/meminfo.go

```diff
@@ -201,13 +273,20 @@ func GetNodeNUMAInfo() (*NodeNUMAInfo, error) {
 			klog.V(4).Infof("failed to read NUMA info, dir %s, err: %v", dirName, err)
 			continue
 		}
+		hugepageInfos, err := GetHugePagesInfo(dirName)
```

Comment:
Do we need to abort the whole NUMA info reporting when the huge page collecting fails? Perhaps we can report empty info only for the failed part in that case.